src/pages/export/MetaData.js

```diff
@@ -1,62 +1,65 @@
+import { Form } from 'react-final-form'
+import { Button } from '@dhis2/ui-core'
 import React from 'react'
 import i18n from '@dhis2/d2-i18n'
+import cx from 'classnames'
 
-import { FormBase } from '../../components/FormBase'
+import { FormContent } from '../../components/FormSections/FormContent'
+import { FormFooter } from '../../components/FormSections/FormFooter'
+import { FormHeader } from '../../components/FormSections/FormHeader'
 import { MetadataExportIcon } from '../../components/Icon'
-import { getFormFields, getFormValues, getDownloadUrl } from '../../helpers'
-import { isProduction } from '../../helpers/env'
+import { Schemas } from '../../components/Inputs/Schemas'
+import { Format } from '../../components/Inputs/Format'
+import { Compression } from '../../components/Inputs/Compression'
+import { Sharing } from '../../components/Inputs/Sharing'
+import {
+    EXCLUDE_SCHEMAS,
+    supportedFormats,
+    initialValues,
+    onSubmit,
+} from './MetaData/helper'
+import stylesForm from '../../components/Form/styles.module.css'
+import stylesFormBase from '../../components/FormBase/styles.module.css'
```

Comment:
I'd use formStyles and formBaseStyles here. Otherwise it reads as if it's forms instead of styles.